airflow/operators/bash.py

```diff
@@ -134,13 +137,20 @@ def __init__(
         env: Optional[Dict[str, str]] = None,
         output_encoding: str = 'utf-8',
         skip_exit_code: int = 99,
+        bash_cwd: str = None,
         **kwargs,
     ) -> None:
         super().__init__(**kwargs)
         self.bash_command = bash_command
         self.env = env
         self.output_encoding = output_encoding
         self.skip_exit_code = skip_exit_code
+        self.bash_cwd = bash_cwd
+        if self.bash_cwd is not None:
+            if not os.path.exists(self.bash_cwd):
+                raise AirflowException(f"Can not find the bash_cwd: {bash_cwd}")
+            if not os.path.isdir(self.bash_cwd):
+                raise AirflowException(f"The bash_cwd {bash_cwd} must be a folder")
```

Comment:
```suggestion
                raise AirflowException(f"The bash_cwd {bash_cwd} must be a directory")
```

(The two are slightly different and technically we don’t need it to be a folder.)

Reply:
Got it. 
changing it.